be/src/olap/rowset/beta_rowset.cpp

```diff
@@ -54,11 +54,12 @@ Status BetaRowset::do_load(bool /*use_cache*/) {
     return Status::OK();
 }
 
-Status BetaRowset::load_segments(std::vector<segment_v2::SegmentSharedPtr>* segments) {
+Status BetaRowset::load_segments(std::vector<segment_v2::SegmentSharedPtr>* segments,
+                                 const TabletSchema* read_tablet_schema) {
     for (int seg_id = 0; seg_id < num_segments(); ++seg_id) {
```

Comment:
Why segment should related with the input schema? Actually segment or rowset has its own schema.